### PR TITLE
Update heron cli to support update command for scaling

### PR DESCRIPTION
**Pull request by billonahill**
Last change required to make scaling available by adding the `update` command to the heron client.

Required for #1292.

### CODE REVIEW

File: heron/tools/cli/src/python/update.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    # invoke the runtime manager to kill the topology
+    execute.heron_class(
+        'com.twitter.heron.scheduler.RuntimeManagerMain',
+        lib_jars,
+        extra_jars=[],
+        args=new_args
+    )
+
+  except Exception as ex:
+    print 'Error: %s' % str(ex)
```
Comment (objmagic) [inline heron/tools/cli/src/python/update.py]: maybe use `Log.debug(traceback.format_exc(ex))`?

**Comment by objmagic** [comment]
👍

File: heron/tools/cli/src/python/update.py (hunk 1 above)
Comment (billonahill) [inline heron/tools/cli/src/python/update.py]: good catch, fixed.

**Comment by kramasamy** [comment]
👍

**Pull request merged**